web/src/components/shared/DetermineComponentForRoute.jsx

```diff
@@ -14,12 +14,13 @@ import StepTerraform from "./StepTerraform";
 import KustomizeEmpty from "../kustomize/kustomize_overlay/KustomizeEmpty";
 import KustomizeOverlay from "../../containers/KustomizeOverlay";
 import ConfigOnly from "../../containers/ConfigOnly";
+import { fetchContentForStep } from "../../redux/data/appRoutes/actions";
```

Comment:
I see, yeah my main concern was if it would be something that would potentially be needed elsewhere we should put it at the top and then could pass it down to the components that would maybe need it. I think this is fine based on this explanation. Thanks!